setup.cfg

```diff
@@ -2,6 +2,7 @@
 line_length = 120
 skip_glob = */migrations/*
 known_first_party = django_pain
+known_third_party = lxml
```

Comment:
`default_section = THIRDPARTY` usually has better outcome.

Reply:
Yay! I've been searching for viable alternative, but this one escaped my sight.

Reply:
Fixed.